ros_ws/src/control/include/control/remote_control.h

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file RemoteControl.h
+ * @class RemoteControl
+ * @brief The class which converts joy messages to twist
+ *
+ * This class will act as a way to take controller inputs and output them to a format which is usable by the robot.  It
+ * is currently setup to accept joy messages and output them as a twist message which only sets the linear.x and
+ * angular.z velocities.
+ */
+
+#ifndef CONTROL_INCLUDE_CONTROL_REMOTE_CONTROL_H_
+#define CONTROL_INCLUDE_CONTROL_REMOTE_CONTROL_H_
+
+#include <ros/ros.h>
+#include <sensor_msgs/Joy.h>
+#include <geometry_msgs/Twist.h>
+
+#define Y_AXIS_L_STICK 1
+#define X_AXIS_R_STICK 2
```

Comment:
In more modern use of C++, using #define for constants is generally discouraged. Compiler preprocessor directives are better suited to compilation-related tasks, such as switching out code across different build targets.

I would rather we use constants here as well, unless you had a specific reason for defining those in this way.